doc/changes/latest.inc

```diff
@@ -63,4 +66,4 @@ Bugs
 
 API changes
 ~~~~~~~~~~~
-- Nothing yet
+- :func:`mne.concatenate_raws` now raises an exception if ``raw.info['dev_to_head_t']`` differs between files. This behavior can be controlled using the new ``on_mismatch`` parameter (:gh:`9438` by `Richard Höchenberger`_)
```

Comment:
This is an immediate change in behavior without a deprecation cycle, so it has to live in the BUG section